### PR TITLE
I18nize toolbox category names

### PR DESCRIPTION
**Pull request by Hamms**
Depends on https://github.com/code-dot-org/code-dot-org/pull/22360

Adds our first instance of block parsing to i18n!

Note that the way this is set up causes category names to be translated globally, rather than on a level-by-level basis. The resulting new i18n file will look [something like this](https://gist.github.com/Hamms/123ac85bc687d8c5e380d1950eb9604e).

### CODE REVIEW

**Comment by Hamms** [comment]
[Start work to translate toolbox categories](https://trello.com/c/qYrUdyQo/102-start-work-to-translate-toolbox-categories)

File: bin/i18n/sync-codeorg-in.rb [hunk 1]
```diff
@@ -44,43 +44,56 @@ def localize_level_content
...
   Dir.glob("dashboard/config/scripts/levels/*.level").sort.each do |file|
     level_name = File.basename(file, ".*")
     File.open(file) do |data|
       level_xml = Nokogiri::XML(data, &:noblanks)
+
+      # Properties
       config = JSON.parse(level_xml.xpath('//../config').first.text)
 
-      # Instructions
+      ## Instructions
```
Comment (joshlory) [inline bin/i18n/sync-codeorg-in.rb:57]: Intentional?

**Comment by joshlory** [comment]
This is just the "in" part, yeah?  Makes sense for now — but what does the "out" merge process look like?

File: bin/i18n/sync-codeorg-in.rb (hunk 1 above)
Comment (Hamms) [reply bin/i18n/sync-codeorg-in.rb:57]: Yeah, I was intending this to be a "sub-heading" under the "Properties" heading, with "Blocks" as a new second top-level heading.

If that's unclear or confusing, though, I can undo it

Comment (joshlory) [reply bin/i18n/sync-codeorg-in.rb:57]: It's just a Ruby comment, yeah?  No strong preference — my initial reaction was that I don't know what it is 😛.

**Pull request merged**